strong-config/node
# chore(example project): use partially encrypted config with PGP setup

Donezo:

- [x] Use partially encrypted example config instead of no encryption at all
- [x] Provide example PGP keypair (freshly generated) to perform the encryption<->decryption

How to test:

1. Install `gpg`, e.g. with `brew install gpg`
2. Import PGP key via `gpg --import example/pgp/example-keypair.pgp`
3. Now, `yarn dev:load` should just work and output the decrypted config value

This should be mentioned in the README. Depending on which PR goes in first (#29 or this here), we should update the PR going second accordingly.

Facilitates #26
